.buildkite/pipeline.yml

```diff
@@ -21,7 +21,7 @@ steps:
       # that occurs the first time after cleaning if one of your dependences uses a binary framework
       # See https://bugs.swift.org/browse/SR-13803
 
-      build_and_test_pod 'ios test' || build_and_test_pod 'ios test'
+      build_and_test_pod ios test || build_and_test_pod ios test
```

Comment:
Good catch remembering to update those while updating from bash-cache `1.x` to `2.x` 👏

Reply:
Thanks! But really... CI failed, I didn't remember 🙃 But I'll take it anyway 💪

Reply:
🤫 (I won't tell, promise 🤐) 😜